ReSharper.FSharp/src/FSharp.Psi.Features/src/QuickFixes/ReplaceWithAssignmentExpressionFix.fs

```diff
@@ -0,0 +1,55 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+
+open FSharp.Compiler.SourceCodeServices
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.Highlightings.CommonErrors
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Parsing
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Resources.Shell
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Util
+open JetBrains.ReSharper.Psi.ExtensionsAPI
+open JetBrains.ReSharper.Plugins.FSharp.Util
+open JetBrains.ReSharper.Psi
+
+type ReplaceWithAssignmentExpressionFix(warning: UnitTypeExpectedWarning) =
+    inherit FSharpQuickFixBase()
+   
+    let expr = warning.Expr.As<IBinaryAppExpr>()
+    override x.IsAvailable _ =
```

Comment:
Code style nit-picking: would be great to have an empty line between declarations when at least one of them is multi-line.